sa/database.go

```diff
@@ -12,11 +13,34 @@ import (
 	blog "github.com/letsencrypt/boulder/log"
 )
 
+// DbSettings contains tuning setting for the sql driver. Optional
+// fields should only be applied if their zero value matches the default
+// of the sql driver. SA is the authoritative closer of connections when
+// ConnMaxIdleTime and ConnMaxLifetime are set lower than their mariadb
+// counterparts interactive_timeout and wait_timeout.
```

Comment:
I don't think there's a concept of "authoritative closer," just "closer."

Also I would rephrase this in terms of a recommendation: "ConnMaxIdleTime and ConnMaxLifetime should be set lower than their mariab counterparts interactive_timeout and wait_timeout. Since closing connections incurs a small overhead, we'd like to spread that overhead across the SAs rather than making the database do it."